src/lints/unwind_function_missing.ron

```diff
@@ -0,0 +1,51 @@
+SemverQuery(
+    id: "unwind_function_missing",
+    human_readable_name: "unwind function missing",
+    description: "pub fn changed from an unwind-capable ABI to the same-named ABI without unwind: \"C-unwind\" to \"C\"",
+    required_update: Major,
+    reference_link: None,
+    query: r#"
+    {
+        CrateDiff {
+            baseline {
+                item {
+                    ... on Function {
+                        visibility_limit @filter(op: "=", value: ["$public"])
+
+                        abi_: abi {
+                            name @tag
+                            raw_name @output
+                            unwind @filter(op: "=", value: ["$true"])
+                        }
+                    }
+                }
+            }
+            current {
+                item {
+                    ... on Function {
+                        visibility_limit @filter(op: "=", value: ["$public"])
+
+                        name @output
+
+                        new_abi_: abi {
+                            name @filter(op: "=", value: ["%name"])
+                            raw_name @output
+                            unwind @filter(op: "!=", value: ["$true"])
+                        }
+
+                        span_: span @optional {
+                            filename @output
+                            begin_line @output
+                        }
+                    }
+                }
+            }
+        }
+    }"#,
+    arguments: {
+        "public": "public",
+        "true": true,
+    },
+    error_message: "An unwind-capable ABI changed to same-named ABI without unwind",
+    per_result_error_template: Some("pub fn {{name}} changed ABI from {{abi_raw_name}} to {{new_abi_raw_name}} in {{span_filename}}:{{span_begin_line}}"),
+)
```

Comment:
Please ensure all files end in a trailing newline — you should be able to configure your editor to do this automatically for you.
```suggestion
)

```

Reply:
Got it, thanks! Just changed my editor configuration.